android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -212,33 +203,47 @@ synchronized public void setNowPlaying(ReadableMap metadata) {
         nb.setContentInfo(album);
         nb.setColor(notificationColor);
 
-        if(artwork != null) {
-            final String artworkUrl = artwork;
-            final boolean artworkLocal = localArtwork;
-
-            artworkThread = new Thread(new Runnable() {
-                @Override
-                public void run() {
-                    Bitmap bitmap = loadArtwork(artworkUrl, artworkLocal);
-                    md.putBitmap(MediaMetadataCompat.METADATA_KEY_ART, bitmap);
-                    nb.setLargeIcon(bitmap);
-
-                    session.setMetadata(md.build());
-                    notification.show(nb, isPlaying);
-                    artworkThread = null;
-                }
-            });
-            artworkThread.start();
+        if (metadata.hasKey("artwork")) {
```

Comment:
Can you keep the same code style throughout the code?
This `if` should not have a space after